schema/addresses/address.yaml

```diff
@@ -0,0 +1,63 @@
+---
+"$schema": https://json-schema.org/draft/2020-12/schema
+title: address
+description:
+  Address points
+type: object
+properties:     # JSON Schema: Top-level object properties.
+  id: { "$ref": ../defs.yaml#/$defs/propertyDefinitions/id }
+  geometry:
+    description:
+      Division geometry MUST be a Point as defined by GeoJSON schema.
+      It represents the approximate location of a position commonly
+      associated with the real-world entity modeled by the division
+      feature.
+    unevaluatedProperties: false
+    allOf:
+      - "$ref": https://geojson.org/schema/Point.json
+  properties:   # GeoJSON: top-level object 'properties' property.
+    unevaluatedProperties: false
+    allOf:
+      - "$ref": ../defs.yaml#/$defs/propertyContainers/overtureFeaturePropertiesContainer
+    properties: # JSON Schema: properties within GeoJSON top-level object 'properties' property
+      country:
+        description: The country code in the Open Addresses distribution
```

Comment:
Does "in the OpenAddresses distribution" add value? Can we just say it's the ISO-3166-1 alpha-2 country code?